lib/asciidoctor/pdf/converter.rb

```diff
@@ -2180,10 +2180,23 @@ def convert_table node
           head_border_bottom_width = theme.table_head_border_bottom_width || table_border_width
         end
         [:top, :bottom, :left, :right].each {|edge| border_width[edge] = table_border_width }
+
         table_grid_color = theme.table_grid_color || table_border_color
-        table_grid_style = (theme.table_grid_style || table_border_style).to_sym
+        table_grid_style = theme.table_grid_style || table_border_style
+        if table_grid_style.is_a? Array
+          raise 'table-grid-style array must contain two styles' if table_grid_style.size != 2
```

Comment:
I prefer not to do validations in the converter, so this check is not necessary. Let's check for an Array and assume the theme gave us what we expect.